### ISSUE TITLE
FreeCS doesn't find listen server on LAN, but can join it through console

### ISSUE THREAD

**Issue by hellodub** [eukara/FreeCS#28]
Starting with a fresh install of FreeHL (4-18-2023) with assets pulled from retail disc, and FreeCS (4-18-2023) with assets downloaded using the install_from_archiveorg.sh script.

Both of these are linux machines, one is Linux Mint 21.1 and the other is Pop_OS!, so they are relatively close Ubuntu-based distros.

I go to Play on LAN -> start a server on one machine, and also go to Play on LAN and choose 'Refresh' to which the prompt : 

'"**Refreshing LAN server list...**" 
**Listing 0 applicable servers**' 

...appears but sits endless printing "**Added 0 LAN servers (of 0).**" to the console.

However, the server is there and can be joined through the console no problem with "connect <hostname>".

Is this a problem with FTEQW perhaps? I'm looking around the Innernette for problems relating to it, but so far have not gleaned much.

**Comment by eukara** [comment]
Yeah, this is a bug within FTEQW. Until recently it was making some pretty bad assumptions also (where it restricted LAN servers to only show up when the game's com_protocolname was 'Quake'). This will be resolved within FTEQW.
